example/src/main/java/andrewoid/neuralnetwork/MNISTReader/IdxReader.java

```diff
@@ -0,0 +1,112 @@
+package andrewoid.neuralnetwork.MNISTReader;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class IdxReader
```

Comment:
If you copied this code from somewhere, include a link to it in a javadoc comment to the class.